src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -28,7 +28,7 @@ protected void sendMetricPoint(String metricName, double value, String[] tags) {
             this.statsDClient.stop();
             init();
         }
-        statsDClient.gauge(metricName, value, tags);
+        statsDClient.gauge(StatsdReporter.formatDataName(metricName), value, tags);
```

Comment:
Yeah, I know that. The problem is that the service check name isn't defined in user configurations :/

My point here was just to enable us to stop getting the service check prefix from the configuration file and therefore being able to have versioned configuration file without getting versioned prefixes in our backend (activemq.yaml, activemq_58.yaml, activemq_62.yaml...). In the first place I just did a dirty `if(checkName == 'activemq_58') checkName = 'activemq';` do you think this would be more suitable ? Or maybe we could format the metricName when it's read from the yaml file name instead of doing it every time the metrics'service checks are sent.